pyproject.toml

```diff
@@ -42,6 +42,7 @@ numpy = [
   { version = ">=2.1", python = "^3.13" }
 ]
 hightime = { version = ">=0.2.2", allow-prereleases = true }
+typing_extensions = { version = ">=4.0.0" }
```

Comment:
This package **did** depend on typing-extensions: https://github.com/ni/nitypes-python/blob/0.1.0-dev7/pyproject.toml

@mjohanse-emr deleted it in https://github.com/ni/nitypes-python/pull/190 and I didn't notice.

Was that intentional or did the Poetry migration plugin do it automatically?